backend/controllers/project.controller.js

```diff
@@ -67,4 +67,26 @@ ProjectController.destroy = async function (req, res) {
 };
 
 
+ProjectController.getProjectManagers = async function (req, res) {
+  try {
+    const userProjectMap = {};
```

Comment:
I think we want an array of objects of shape:

``` js
const userProjectMap = []; //changed to array
```

```js
// Object shape from API call
{
    user, // will this just be the userId string or will
          // we have a populated user object with email, etc?
    project: {
        name: string
        projectId: string
    }
}
```

Does that look right to you? Then we'd be returning an array that can be sorted more easily on the front end with some sort functions